guides/source/action_cable_overview.md

```diff
@@ -212,11 +213,6 @@ function getWebSocketURL {
 A consumer becomes a subscriber by creating a subscription to a given channel:
 
 ```js
-// app/javascript/channels/chat_channel.js
-import consumer from "./consumer"
-
-consumer.subscriptions.create({ channel: "ChatChannel", room: "Best Room" })
-
```

Comment:
You've deleted this code example, but weren't you planning to replace it with something?